### DISCUSSION TITLE
Meteor M2 reception

### DISCUSSION THREAD

**Discussion by ShianMull** [jekhokie/raspberry-noaa-v2#547] [Q&A] [answered]
Hi, I have just setup Rpi NOAA v2 with RTL-SDR V3 connected to a cross dipole aerial used on my old MapSat setup.  I’m getting some lovely images from the NOAA satellites, but decoding nothing from Meteor M2.  I’ve tried gnuradio and rtl-fm decoding but nothing appears in output (not even a pure noise frame).  I’ve read that there have been issues with Meteor, but I’m assuming M2 is transmitting and it’s just the software not decoding - I couldn’t see any similar solutions in other discussions, but please forgive a newbie if I’ve missed anything.  Has anyone seen this before and can offer a pointer to get me going?

Incidentally, just as user feedback, auto gain works best on my cross dipole setup - fixed gain settings did not give consistently best results.

All the best

mark

**Comment by dom-robinson** [comment, marked as answer]
There are currently lots of issues with M2 - 'its unlikely to be you' :) - The satellite has been in sync with noaa19 for a few days, but has had other issues - id wait a little while before you do any thing...!

**Reply by ShianMull** [reply]
Thanks Dom, I have seen a very thin sliver of Meteor M2 data just yesterday, so likely not me as you note!  Cheers, Mark
